README.md

```diff
@@ -32,6 +32,8 @@ Method    | Path
 **GET**   | `/company/{companyNumber}/corporate-body-appointments/persons-of-significant-control`| Calls service to retrieve the psc data for the company
 **GET**   | `/company/{companyNumber}/confirmation-statement/paid`                               | Calls service to verify confirmation statement payment has been made for company on due date
 **GET**   | `/company/{companyNumber}/register/location`                                         | Calls service to retrieve the registers locations for a company
+**GET**   | `/managing-officer/{incorporationNumber}/data`                                       | Calls service to retrieve the managing officer data 
```

Comment:
change incorporationNumber to companyNumber everywhere

Reply:
Please make the endpoint overseas-entity/managing-officer - i.e. specific to overseas entities